server/src/server.ts

```diff
@@ -3,18 +3,25 @@ import type { TwilioCredentials } from './common/types';
 import { createTokenRoute } from './routes/token';
 import { createTwimlRoute } from './routes/twiml';
 import { createLogMiddleware } from './middlewares/log';
+import { auth } from 'express-oauth2-jwt-bearer';
+import config from '../config';
 
 export function createExpressApp(twilioCredentials: TwilioCredentials) {
```

Comment:
We should think of some way to incorporate the auth configs into a similar mechanism. Maybe we rename from `twilioCredentials` to `serverConfig` or something similar, but I don't think we should bake configs into more than one file or mechanism (env vars).